targets/DEV1-BMS/main.cpp

```diff
@@ -198,7 +198,7 @@ int main() {
         // Handle executing timer events that have elapsed
         COTmrProcess(&canNode.Tmr);
         // Update the state of the BMS
-        bms.process();
+        bms.canTest();
```

Comment:
Should this be canTest or should it be process? It is not in the testing file so I feel like it should be process.

Reply:
Yeah, that's my bad. Forgot to remove after testing